build-netlite.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+usage() {                                                                                                                  
+        echo "usage: $(basename "$0") [-h|--help] <clients_version>"                  
+}                                                                                                                          
+                                                                                                                           
+if [ $# -lt 1 ] || [[ "$@" = *"-h"* ]]; then                                                                               
+        usage                                                                                                              
+        exit 0                                                                                                             
+fi                                                                                                                         
+                                                                                                                                                                                                  
+CLI_VER=$1                                                                                                                 
```

Comment:
this could be confusing. the script does not in any way enforce that this is really the client version that is used

Reply:
This way, it is better for automation as it creates a uniquely named directory and tarball that is then uploaded to artifactory. What do you suggest? Remove the compressing part along with CLI_VER and compress it within the job itself?